extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -114,17 +121,21 @@ export class AmpAdXOriginIframeHandler {
         this.iframe, 'send-embed-state', true,
         () => this.sendEmbedInfo_(this.baseInstance_.isInViewport()));
 
-    // To provide position to inabox.
-    if (isExperimentOn(this.win_, 'inabox-position-api')) {
+    if (getExperimentBranch(
```

Comment:
Let's add a TODO here that ideally we would turn this on if A4A.isXhrEnabled() returns false OR if we just decide to turn it on for all traffic (e.g. inabox-position-api is set to true)

Reply:
done